packages/launchpad/src/migration/ConvertConfigFile.vue

```diff
@@ -5,11 +5,11 @@
       <MigrationListItem>
         {{ t("migration.configFile.changedDefault") }}
```

Comment:
When I tried to pass in a custom config file, the screen looks like this:

<img width="1201" alt="image" src="https://user-images.githubusercontent.com/4873279/157278058-47c95b35-d5c6-44e2-a94e-09461c7f065e.png">

The specific wording is: "We've changed the default Cypress config file from: cypress-blah.json -> cypress-blah.config.js"  I'm wondering if the wording needs to be tweaked if we're using a custom config file since we're not in fact using the default Cypress config file.